### PR TITLE
smtp: add raw extraction test

### PR DESCRIPTION
**Pull request by maurizioabba**
pcap downloaded from https://osqa-ask.wireshark.org/questions/33094/extract-an-attachment-email-smtp-cap

### CODE REVIEW

File: tests/smtp-raw-extraction/suricata.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+%YAML 1.1
+---
+
+include: ../../etc/suricata-4.0.3.yaml
```
Comment (victorjulien) [inline tests/smtp-raw-extraction/suricata.yaml]: this should not be necessary (in fact I think we should remove it from the instructions)

File: tests/smtp-raw-extraction/test.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+requires:
+  features:
+    - HAVE_NSS
+  files:
+    - src/output-filestore.c
+
```
Comment (victorjulien) [inline tests/smtp-raw-extraction/test.yaml]: think we need a 'min-version: 5.0.0' here in the requires, otherwise this will break for 4.0 and 4.1

**Comment by victorjulien** [comment]
Can you add a README with a link to the pcap source?

**Comment by maurizioabba** [comment]
Replaced by https://github.com/OISF/suricata-verify/pull/11

**Pull request closed without merging**